packages/terra-site/src/examples/form-textarea/TextareaExamples.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
I know this is a bit late to the game, but we should update this example to also be able to render isInvalid state. Possibly when the input length is 0.

Reply:
Got it covered on the second one actually! https://github.com/cerner/terra-core/pull/982/files#diff-9a8da08c4bb9abc6d8eb06fa7f4dc9a9R39

Keep in mind that default doesn't have errored out styles, so you won't see them on terra-core. They are in the consumer theme though.